.ci/magician/cmd/test_eap_vcr.go

```diff
@@ -141,25 +141,20 @@ func execTestEAPVCR(changeNumber, genPath, kokoroArtifactsDir, modifiedFilePath
 	for s := range services {
 		servicesArr = append(servicesArr, s)
 	}
-	analyticsData := analytics{
-		ReplayingResult:  replayingResult,
+	postReplayData := postReplay{
 		RunFullVCR:       runFullVCR,
 		AffectedServices: sort.StringSlice(servicesArr),
-	}
-	testsAnalyticsComment, err := formatTestsAnalytics(analyticsData)
-	if err != nil {
-		return fmt.Errorf("error formatting test_analytics comment: %w", err)
+		ReplayingResult:  replayingResult,
+		ReplayingErr:     replayingErr,
+		LogBucket:        "ci-vcr-logs",
+		Version:          provider.Private.String(),
+		Head:             head,
 	}
 	if len(replayingResult.FailedTests) > 0 {
-		withReplayFailedTestsData := withReplayFailedTests{
-			ReplayingResult: replayingResult,
-		}
-
-		withReplayFailedTestsComment, err := formatWithReplayFailedTests(withReplayFailedTestsData)
+		comment, err := formatPostReplay(postReplayData)
```

Comment:
Formatting & posting the comment can happen before this if statement, since it's now the same render + post either way.